core/src/main/java/org/apache/accumulo/core/tabletserver/log/LogEntry.java

```diff
@@ -76,12 +78,12 @@ private static void validateFilePath(String filePath) {
   }
 
   /**
-   * Make a copy of this LogEntry but replace the file path.
+   * Add LogEntry information to the provided mutation.
    *
-   * @param filePath path to use
+   * @param mutation the mutation to update
    */
-  public LogEntry switchFile(String filePath) {
-    return new LogEntry(filePath);
+  public void addToMutation(Mutation mutation) {
```

Comment:
Could  update the following to use this new method

https://github.com/apache/accumulo/blob/9701ccface1f94b8d4827323b540ddda9d0cd6c6/server/base/src/main/java/org/apache/accumulo/server/metadata/TabletMutatorBase.java#L178